### DISCUSSION TITLE
Getting started with this new version

### DISCUSSION THREAD

**Discussion by alanoatwork** [areed1192/td-ameritrade-api#9] [Q&A]
I'm currently using the older version successfully but thought I would convert my code to the newer codebase. 

So, I copied the example given in the instructions and the library doesn't even load correctly, I get "Cannot find reference 'credentials' in 'imported module td'.

Is version 0.1.1 ready for primetime?

**Comment by granthooks** [comment]
Same issue here. 
I've tried the following:
-created new py3.8 virtual env in Anaconda
-installed dependencies in requirements.txt and pip installed py-tda-api.
However in VS Code still can't find credentials in module 'td'.

**Reply by usapkota** [reply]
It works for me, make sure you run the setup file as instructed.

**Comment by pprivon** [comment]
I created a new environment in conda with python 3.8 and followed the requirements.txt and pip install py-tda-api. I ran the use_ouath.py code (I know there is a typo in the filename) and all of the dependencies/classes/functions were recognized but erroring out at the TDA authentication page. I confirmed my client_id and redirect_uri match my app on TDA's website and I have the config.ini file in the proper location. Looking to confirm if anyone is successful with the initial authentication token retrieval with this code base or only using tokens they already have. Thanks in advance.
